AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSCosmosDb.m

```diff
@@ -0,0 +1,136 @@
+#import "MSCosmosDb.h"
+#import "AppCenter+Internal.h"
+#import "MSCosmosDbIngestion.h"
+#import "MSDataStorageInternal.h"
+#import "MSTokenResult.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Document DB base endpoint.
+ */
+static NSString *const kMSDocumentDbEndpoint = @"https://%@.documents.azure.com";
```

Comment:
Looking at all these constants makes me think that it's a good idea to move them to their own `MSDataSyncConstants.h` like we do for analytics in `MSAnalyticsConstants.h`.

Reply:
I like idea of creating a .h file that hold the consts, however, that should be used for common const only. it's good practice to keep the consts pertinent to a class consuming it (readability and clarity about intention of the const + easier to cleanup)